libs/@local/telemetry/src/logging/otlp.rs

```diff
@@ -0,0 +1,40 @@
+use error_stack::Report;
+use opentelemetry_appender_tracing::layer::OpenTelemetryTracingBridge;
+use opentelemetry_otlp::{ExporterBuildError, WithExportConfig as _};
+use opentelemetry_sdk::{Resource, logs::SdkLoggerProvider};
+use tracing::Subscriber;
+use tracing_subscriber::{Layer, registry::LookupSpan};
+
+use crate::OtlpConfig;
+
+pub(crate) fn provider(
+    config: &OtlpConfig,
+) -> Result<Option<SdkLoggerProvider>, Report<ExporterBuildError>> {
+    let Some(endpoint) = config.logs_endpoint.as_deref() else {
+        return Ok(None);
+    };
+
+    Ok(Some(
+        SdkLoggerProvider::builder()
+            .with_resource(Resource::builder().with_service_name("graph").build())
```

Comment:
shouldn't the service name be configurable? e.g. the type-fetcher should have a different name than the graph. No?